src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,133 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { getSearchWith } from '../utils/searchHelper';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [loadError, setLoadError] = useState(false);
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sort = searchParams.get('sort');
+  const isReversed = searchParams.get('order') === 'desc';
+
+  let visiblePeople = people.filter(person => {
+    return person.name.toLowerCase().includes(query.toLowerCase().trim())
+      || person.fatherName?.toLowerCase().includes(query.toLowerCase().trim())
+      || person.motherName?.toLowerCase().includes(query.toLowerCase().trim());
```

Comment:
Repeatable code `?.toLowerCase().includes(query.toLowerCase().trim()` move to helper function and reuse it